changes to src/runtime/stream/stream.c breaks compilation.

On the latest version of `development` compilation breaks with the following error message:

```
==== Building stream (debug) ====
Creating obj/debug/Debug/stream
stream.c
../stream/stream.c:44:22: error: static declaration of 'scons_mk' follows non-static declaration
 static struct scons *scons_mk(pony_ctx_t *ctx, pony_type_t *type){
```

--- a/src/runtime/stream/stream.c
+++ b/src/runtime/stream/stream.c
@@ -41,7 +41,7 @@ void stream_trace(pony_ctx_t *ctx, void *p)
   future_trace(ctx, p);
 }
 
-static struct scons *scons_mk(pony_ctx_t *ctx, pony_type_t *type){
+struct scons *scons_mk(pony_ctx_t *ctx, pony_type_t *type){
   struct scons *scons = pony_alloc(ctx, sizeof(struct scons));
   scons->eos = false;
   scons->element = (value_t) {.p = NULL};

